docker/Dockerfile.development

```diff
@@ -25,7 +25,7 @@ COPY . /home/$USER_NAME/mgmt
 WORKDIR /home/$USER_NAME/mgmt
 
 # Install dependencies
-RUN make deps
+RUN make dev
```

Comment:
Assuming you want to change this because you want `bindata`, we don't, the bindata target is a `build` Makefile dep.